src/go/rdctl/cmd/snapshotDelete.go

```diff
@@ -24,13 +25,16 @@ func init() {
 }
 
 func deleteSnapshot(cmd *cobra.Command, args []string) error {
-	cmd.SilenceUsage = true
-	paths, err := p.GetPaths()
+	appPaths, err := paths.GetPaths()
 	if err != nil {
 		return fmt.Errorf("failed to get paths: %w", err)
 	}
-	manager := snapshot.NewManager(paths)
-	if err = manager.Delete(args[0]); err != nil {
+	manager := snapshot.NewManager(appPaths)
+	id, err := getSnapshotId(manager, args[0])
+	if err != nil {
+		return fmt.Errorf("can't delete snapshot: %w", err)
```

Comment:
It is not that you can't delete it, you can't find it:

```suggestion
		return fmt.Errorf("can't locate snapshot: %w", err)
```

Reply:
In that case I might as well just return the error message returned by `GetSnapshotId` since there's no point adding more information